build/config.js

```diff
@@ -72,10 +81,9 @@ const genConfig = (name) => {
             replace({
                 __WEEX__: false,
                 __VERSION__: VueVersion,
-                'process.env.NODE_ENV': "'development'",
```

Comment:
@farfromrefug Does this remove all the development related help messages that vue prints? If so - can this be re-enabled while developing?

Reply:
well it s the user choice by replacing within his weback config. that code was making the appear always, even in prod

Reply:
@farfromrefug alright, so as far as I understand this change, this should still allow the help messages to appear unless webpack sets the env to "production"?

Reply:
@rigor789 indeed it will appear by default because it checks for ```NODE_ENV !== 'production'``` which is the case as ```NODE_ENV``` is not defined. 
However that change allow the user tu define ```NODE_ENV``` to ```production``` when building for production, and thus remove all the warning. 
It is just moved to userland